PS4Bot.py

```diff
@@ -384,6 +386,8 @@ def handle_command(self, message, command, rest):
             self.join_or_bail(message, rest, bail = True)
         elif command == "scuttle":
             self.maybe_scuttle_game(message, rest)
+        elif command == "thanks" or command == "thankyou" or command == "thank you" or command == "ta" or command == "cheers":
```

Comment:
Noyce, although `"thank you"` will never come through as a command because we split on whitespace, if you could take that one out (or allow `command == "thank" and rest == "you"`, but i don't think anyone's used "thank you" yet)

Reply:
Done